src/ngSanitize/filter/linky.js

```diff
@@ -137,8 +163,21 @@ angular.module('ngSanitize').filter('linky', ['$sanitize', function($sanitize) {
     }
 
     function addLink(url, text) {
+      var key;
+      var computedAttributes = {};
       html.push('<a ');
-      if (angular.isDefined(target)) {
+      if (angular.isObject(attributes)) {
```

Comment:
I guess that's a good idea. At first I didn't want to reassign the `attributes` variable, but the extra check for the return value of attributes as a function, is a compelling case for doing it like this.